x-pack/plugins/enterprise_search/public/applications/workplace_search/utils/to_sentence_serial.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AND } from '../views/content_sources/constants';
+
+export const toSentenceSerial = (array: string[]) =>
+  array.length === 1
+    ? `${array}`
```

Comment:
No need to use a template literal here.

```JavaScript
 array.length === 1
    ? array
```

Reply:
We need to convert the array to string. If we return an array, we get:
```
Expected: "One"
Received: ["One"]
```

But I see how it is confusing, so hopefully, this commit should add clarity: https://github.com/elastic/kibana/pull/91555/commits/026560875f034694d7800ef457ccbd4ec170ddb3